sdk/loadtestservice/Azure.Developer.LoadTesting/tests/LoadTestRunClientTest.cs

```diff
@@ -0,0 +1,324 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Core.TestFramework.Models;
+using Azure.Developer.LoadTesting.Tests.Helper;
+using NUnit.Framework;
+
+namespace Azure.Developer.LoadTesting.Tests
+{
+    internal class LoadTestRunClientTest: RecordedTestBase<LoadTestingClientTestEnvironment>
+    {
+        private string _testId;
+        private string _testRunId;
+        private string _fileName;
+        private string _resourceId;
+        private TestHelper _testHelper;
+        private LoadTestAdministrationClient _loadTestAdministrationClient;
+
+        public LoadTestRunClientTest(bool isAsync) : base(isAsync)
+        {
+            _testId = "test-from-csharp-sdk-testing-framework";
+            _fileName = "sample.jmx";
+            _testRunId = "test-run-from-csharp-sdk-testing-framework";
+            _testHelper = new TestHelper();
+
+            BodyRegexSanitizers.Add(new BodyRegexSanitizer(@"sig=(?<group>.*?)(?=\s+)", SanitizeValue)
+            {
+                GroupForReplace = "group"
+            });
+        }
```

Comment:
Should we create a `LoasdTestTestsBase` class and de-dupe this setup from the other test classes?

Reply:
Implemented this approach in the latest commit.